src/sdk/python/examples/basic/run.py

```diff
@@ -9,24 +9,23 @@
 ##############################################################################
 
 import asyncio
-import json
-import os
 import logging
 
+from iotea.core.protocol_gateway import ProtocolGateway
 from iotea.core.util.logger import Logger
+from iotea.core.util.mqtt_client import MqttProtocolAdapter
 from iotea.core.util.talent_io import TalentInput
+
 logging.setLoggerClass(Logger)
 logging.getLogger().setLevel(logging.INFO)
 
-os.environ['MQTT_TOPIC_NS'] = 'iotea/'
-
 # pylint: disable=wrong-import-position
 from iotea.core.talent import Talent
 from iotea.core.rules import AndRules, Rule, ChangeConstraint, Constraint
 
 class MyTalent(Talent):
-    def __init__(self, connection_string):
-        super(MyTalent, self).__init__('python-basic-talent', connection_string)
+    def __init__(self, protocol_gateway_config):
+        super(MyTalent, self).__init__('python-basic-talent', protocol_gateway_config)
```

Comment:
I think it should be `super().__init__` (to be aligned with the other python examples)